packages/cli-kit/src/public/node/cli.ts

```diff
@@ -62,8 +62,8 @@ export async function runCLI(options: RunCLIOptions): Promise<void> {
   run(undefined, options.moduleURL)
     .then(() => flush())
     .then(printEventsJson)
-    .catch((error) => {
-      errorHandler(error)
+    .catch(async (error) => {
```

Comment:
Does this work? I'm not sure you can pass an async function to `catch`, it won't be awaited.

Reply:
If we want to await here we'll need to convert this to a `try/catch` block so something like:

```
try {
   await run(undefined, options.moduleURL)
   await flush()
   printEventsJson()
  } catch {
   await errorHandler(error)
   return Errors.handle(error)
}
```

Reply:
I updated this bit, please let me know what you think! I don't love the `// eslint-disable-next-line no-catch-all/no-catch-all` eslint ignore I had to use, but a search on the CLI repo seems like we do this quite often because we don't want to rethrow errors and interrupt the process, we want to log them instead?